Carrot-Assistant/app.py

```diff
@@ -271,6 +271,22 @@ async def run_db(request: PipelineRequest) -> List[Dict[str,Any]]:
     
 @app.post("/run_vector_search")
 async def run_vector_search(request: PipelineRequest):
+    """
+    Search a vector database for a medication name
+
+    Default options can be overridden by pipeline_options
+    A warning: if you don't have a vector database set up under the embeddings_path, it will build one. This takes a while, an hour on a 2019 macbook pro.
```

Comment:
instead of it will build one, shall we say: this method (or this module will build one for you?) instead of saying macbook pro, shall we just add the cpu/memory specs here?